doc/user_doc/vic_admin/delete_vch_options.md

```diff
@@ -43,6 +43,21 @@ Wrap the password in single quotation marks (') on Mac OS and Linux and in doubl
 
 <pre>--password '<i>esxi_host_or_vcenter_server_p@ssword</i>'</pre>
 
+### `thumbprint` ###
+
+Short name: None
+
+The thumbprint of the vCenter Server or ESXi host certificate. Specify this option if your vSphere environment uses untrusted, self-signed certificates. Alternatively, specifying the `--force` option allows you to omit the `--thumbprint` option. If your vSphere environment uses trusted certificates that are signed by a known Certificate Authority (CA), you do not need to specify the `--thumbprint` option.
+
+To obtain the thumbprint of the vCenter Server or ESXi host certificate, run `vic-machine delete` without the specifying the `--thumbprint` or `--force` options. The deletion of the virtual container host fails, but the resulting error message includes the required certificate thumbprint. 
+
+<pre>Failed to verify certificate for target=<i>vcenter_or_esxi_host</i> (thumbprint=<i>thumbprint</i>)
```

Comment:
Decided to delete the error message and be less specific. I think that we might need a troubleshooting topic about this, that includes all possible error messages. Filed https://github.com/vmware/vic/issues/2996 to track this for GA.